snippets/copyToClipboard.md

```diff
@@ -0,0 +1,23 @@
+### copyToClipboard
+
+Copy a string to the clipboard.
```

Comment:
You should add a note saying it only works as a result of user action (i.e. inside a `click` event listener)

Reply:
That's only on Firefox (works on Chrome regardless of that), but I really should! 👍